### PR TITLE
implementing reflect metadata for fetching component metadata

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

File: src/backend/adapters/angular2.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
 declare var ng: { probe: Function, coreTokens: any };
 declare var getAllAngularRootElements: Function;
+declare var Reflect: { getOwnMetadata: Function };
```
Comment (igor-ka) [inline src/backend/adapters/angular2.ts]: @sumitarora are there typings for reflect-metadata?

Comment (sumitarora) [inline src/backend/adapters/angular2.ts]: not yet

Comment (igor-ka) [inline src/backend/adapters/angular2.ts]: :+1:

**Comment by vanessayuenn** [comment]
lgtm

**Pull request merged**